src/App.tsx

```diff
@@ -1,14 +1,54 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
-import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { TodoModal } from './components/TodoModal';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [loading, setLoading] = useState(true);
+  const [selectedStatus, setSelectedStatus] = useState('all');
```

Comment:
create enum for it, for ex.:
```ts
enum FilterBy {
  ALL = 'all',
  COMPLETED = 'completed',
  ACTIVE = 'active'
}
```
and use it like this:
```suggestion
  const [selectedStatus, setSelectedStatus] = useState(FilterBy.ALL);
```